definitions/sanitize-html_v1.x.x/flow_>=v0.23.x/sanitize-html_v1.x.x.js

```diff
@@ -0,0 +1,23 @@
+type SanitizeHtml$Frame = {
```

Comment:
I hadn't thought of this until just recently, but since you have an `exports` declaration in the module I think you can just move these inside the `declare module` and just drop the namespaces on these.

(might make things more readable? I'll leave it up to you...)